### DISCUSSION TITLE
When I use VS Code and `Run Cell` the `parms` as defined in the yaml frontmatter are not available?

### DISCUSSION THREAD

**Discussion by rkrug** [quarto-dev/quarto-cli#5825] [Q&A] [answered]
### Description

I am using VS Code.

When using parameter in the yaml frontmatter, the document can be rendered, but when execute the individual cell using `Run Cell` the value for `params` is not available.

```
r$> cat(params$value)
Error in cat(params$value) : object 'params' not found
```

What am I missing here?

````cmd
---
title: "Untitled"
format: html
params:
    value: 13
---

```{r}
cat(params$value)
```
````

**Comment by mcanouil** [comment]
Nothing.

This is a feature not available in Quarto VSCode extension, nor in R VSCode extension.

There was an opened feature request that was closed automatically ...
See <https://github.com/REditorSupport/vscode-R/issues/504>.

**Comment by rkrug** [comment]
Hm. I really miss this... Any work arounds, except of the one suggested at https://github.com/REditorSupport/vscode-R/issues/504#issuecomment-1578298924?

**Reply by mcanouil** [reply, marked as answer]
Feel free to open a "formal" feature request for Quarto VSCode extension, see <https://github.com/quarto-dev/quarto/issues>.

**Reply by rkrug** [reply]
Done here [#200](https://github.com/quarto-dev/quarto/issues/200).

Thanks
